src/lolex-src.js

```diff
@@ -1,552 +1,606 @@
-/*global global, window*/
```

Comment:
I had to view this file with [whitespace diff turned off](https://github.com/sinonjs/lolex/pull/25/files?w=1) to view the difference. Could you configure your editor to keep the existing indenting settings? 

We already have an [EditorConfig](https://github.com/sinonjs/lolex/blob/master/.editorconfig) file. Perhaps try installing a plugin for your editor that will use this?